docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -40,6 +40,22 @@ At a minimum, ``hive.metastore.uri`` must be configured:
     connector.name=iceberg
     hive.metastore.uri=thrift://localhost:9083
 
+Iceberg configuration properties
+--------------------------------
+
+================================================== ============================================================ ============
+Property Name                                      Description                                                  Default
```

Comment:
Property name

Reply:
Rest all documentation configuration also have this header as "Property Name". Since I'll be changing this to a list table, I think this won't be required, right?